src/main/java/com/hazelcast/cloud/ClientWithSsl.java

```diff
@@ -1,13 +1,11 @@
 package com.hazelcast.cloud;
 
 import java.util.Properties;
-import java.util.Random;
-
 import com.hazelcast.client.HazelcastClient;
 import com.hazelcast.client.config.ClientConfig;
 import com.hazelcast.cloud.jobs.UpperCaseFunction;
-import com.hazelcast.cloud.model.City;
```

Comment:
You removed examples that uses `City` but you didn't remove `City` itself. Why?

Reply:
In order to be synced with documentations, I changed `User` to `City` with suggestion of @mdumandag. I think this also resolves the comment.